packages/form/components/Form.astro

```diff
@@ -18,13 +19,26 @@ const {
 	formGroups = [],
 	theme,
 	showValidationHints = false,
+	validateOnLoad = false,
 	readOnly = false,
 } = Astro.props;
 
 const uid = new ShortUniqueId({ length: 9 });
 const formTheme = theme ?? 'light';
 const formName = Array.isArray(formGroups) ? null : formGroups?.name || null;
 const formId = Array.isArray(formGroups) ? uid() : formGroups?.id || null;
+
+// if `validateOnLoad` prop is true, 
+// it should forced all FormControl to validate on load
+if (validateOnLoad) {
+	if (Array.isArray(formGroups)) {
+		formGroups.forEach(group =>
```

Comment:
if you loop the `controls` directly here and call `setValidateOnLoad` for each control, I think we can avoid changing the `FormGroup` class . This is preferable for now because we don't have a need for the property `validateOnLoad` in the `FormGroup` class.